src/components/StakingVaults/AllEarnOpportunities.tsx

```diff
@@ -116,7 +116,7 @@ export const AllEarnOpportunities = () => {
       mixpanel?.track(MixPanelEvents.ClickOpportunity, {
         provider,
         type,
-        assets: [getCompositeAssetSymbol(underlyingAssetId ?? '')],
+        assets: [getMaybeCompositeAssetSymbol(underlyingAssetId ?? '')],
```

Comment:
Ditto, should always be defined
```suggestion
        assets: [getMaybeCompositeAssetSymbol(underlyingAssetId)],
```